### PR TITLE
Add whenever gem

### PR DESCRIPTION
**Pull request by aitorlb**
- Add `whenever` gem
- Update `Gemfile.lock`
- Add `schedule.rb`

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# every 2.hours do
+#   command "/usr/bin/some_great_command"
+#   runner "MyModel.some_method"
+#   rake "some:great:rake:task"
+# end
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
```
Comment (isaacmg410) [inline config/schedule.rb]: Remove unecessary examples.

File: config/schedule.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  # Learn more: http://github.com/javan/whenever
+
+  every 1.day, at: '2:00 am' do # 1.minute 1.day 1.week 1.month 1.year is also supported
```
Comment (isaacmg410) [inline config/schedule.rb]: Fix identation please

**Review by isaacmg410 — CHANGES_REQUESTED** [review]
Good Job!

File: config/schedule.rb (hunk 2 above)
Comment (aitorlb) [reply config/schedule.rb]: Thanks

File: config/schedule.rb (hunk 1 above)
Comment (aitorlb) [reply config/schedule.rb]: Thanks

**Review by isaacmg410 — APPROVED** [review]

**Comment by aitorlb** [comment]
Thanks

**Pull request merged**